test/integration/container-runtime/container_runtime.go

```diff
@@ -0,0 +1,255 @@
+// Copyright (c) 2020 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package container_runtime
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"time"
+
+	gardencorev1beta1 "github.com/gardener/gardener/pkg/apis/core/v1beta1"
+	extensionsv1alpha1 "github.com/gardener/gardener/pkg/apis/extensions/v1alpha1"
+	gardenerutils "github.com/gardener/gardener/pkg/utils"
+	"github.com/gardener/gardener/test/framework"
+	"github.com/onsi/ginkgo"
+	g "github.com/onsi/gomega"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const gVisorContainerRuntimeName = "gvisor"
+var gVisorTimeout = 30 * time.Minute
+
+var _ = ginkgo.Describe("gVisor tests", func() {
+	f := framework.NewShootFramework(nil)
+
+	f.Beta().Release().CIt("should add, remove and upgrade worker pool with gVisor", func(ctx context.Context) {
```

Comment:
`Beta` and `Release` together would be strange. `Release` means we'll block gardener delivery if such tests fail, `Beta` means, we want to get a feel for the flakiness of a newly introduced test and not blocking anything for now.
Probably you want either only `Beta()` or `Beta().Serial()` if that test should run exclusively without other tests running concurrently.